samples/keyboard/pkg/pkg.gp4

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>

```

Comment:
File should be moved to sample root directory. eboot.bin should be removed and font.ttf should be in an assets folder to be consistent with other samples.